lms/envs/common.py

```diff
@@ -413,7 +413,7 @@
     'ENABLE_OPENBADGES': False,
 
     # Credit course API
-    'ENABLE_CREDIT_API': False,
+    'ENABLE_CREDIT_API': True,
```

Comment:
I get that this is a hotfix and that we'd rather clean this up later. I've created [ECOM-2482](https://openedx.atlassian.net/browse/ECOM-2482) to track cleanup.